docs/README.md

```diff
@@ -3,10 +3,10 @@
 In `disaster-metadata` => `<year>` => `<disaster>` => `layers` we hold a map of \<asset path : `<data>`>
 
 all `<data>` instances contain the following fields:
-* asset-type `{int}`: corresponds to the LayerType enum
-* display-name `{string}`
-* display-on-load `{boolean}`
-* color-function `{Object}` 
+* assetType `{int}`: corresponds to the LayerType enum
```

Comment:
any interest in converting the docs here into what I did in create_disaster_lib.js: put them with the code?

Reply:
Yeah, do you mind if I do it in a separate PR?